rsyncd/rsyncd.go

```diff
@@ -396,3 +408,14 @@ func (s *Server) Serve(ctx context.Context, ln net.Listener) error {
 		}()
 	}
 }
+
+func validateModule(mod Module) error {
+	if mod.Name == "" {
+		return errors.New("module has no name")
+	}
+	if mod.Path == "" {
+		return fmt.Errorf("module '%s' has empty path", mod.Name)
```

Comment:
use %q instead of '%s' please

Reply:
lol you just taught me what %q means, never used or knew of it before :laughing: